### DISCUSSION TITLE
How to stop pg cluster (scale 0)?

### DISCUSSION THREAD

**Discussion by mahdiG** [cloudnative-pg/cloudnative-pg#2378] [Q&A] [answered]
Hi. How do you stop a pg cluster? The CR doesn't allow `instances: 0` since the minimum is 1. And the operator doesn't use statefulsets to scale the ss to 0. So how can I stop/pause a pg cluster that I've created?

**Comment by phisco** [comment, marked as answer]
You can hibernate the cluster, see: https://cloudnative-pg.io/documentation/1.20/declarative_hibernation/

**Reply by mahdiG** [reply]
Thank you very much! :)

**Reply by mahdiG** [reply]
I used `kubectl annotate cluster <cluster-name> --overwrite cnpg.io/hibernation=on` according to docs but it didn't hibernate the cluster.
It added the annotation to the cluster CR (I tried adding it manually too) but my pods are still running.

I'm using 
chart_version: 0.17.1
cnpg_operator_image_version: 1.19.1
